### PR TITLE
fix awkward Pricing anchor position (fixes #4)

### LINKED ISSUE #4: Anchors jump to awkward positions because of section padding
1. Load https://www.pinecast.com/#pricing
2. 😿 

screen shot 2016-09-19 at 10 53 18 pm

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by cvan** [comment]
### before

screen shot 2016-09-20 at 9 40 52 pm

### after

screen shot 2016-09-20 at 9 39 45 pm

File: css/main.css [hunk 1]
```diff
@@ -188,7 +188,8 @@ header + section p {
 }
 
 .plans-pricing {
-    padding: 100px 15px 0;
+    margin-top: -99px;  /* To adjust for the 100px-tall sticky header. */
```
Comment (cvan) [inline css/main.css]: oh, let me handle mobile width too

**Comment by cvan** [comment]
## mobile

### before

screen shot 2016-09-20 at 9 43 40 pm

screen shot 2016-09-20 at 9 43 53 pm

### after

screen shot 2016-09-20 at 9 43 28 pm

File: css/main.css (hunk 1 above)
Comment (cvan) [reply css/main.css]: done, below

**Comment by mattbasta** [comment]
LOL I was wondering how I would do this, and what you did is exactly what I came up with in my head. I love it.

**Comment by cvan** [comment]
:shucks: 😻

**Pull request merged**